lib/ansible/module_utils/ec2.py

```diff
@@ -58,6 +60,48 @@ class AnsibleAWSError(Exception):
     pass
 
 
+class JSONFileCache(object):
```

Comment:
Could you instead reuse the version from botocore?: https://github.com/boto/botocore/blob/develop/botocore/credentials.py#L182-L224